doc/source/ray-core/ray-dag.rst

```diff
@@ -0,0 +1,155 @@
+.. _ray-dag-guide:
+
+Building Computation Graphs with Ray DAG API
```

Comment:
alternative titles? Declarative Graphs? etc etc...

Reply:
Lazy Ray Computation Graphs?

Reply:
err is current name too long ? I think `computation graph` is quite self-explanatory and we can shorten it to `Computation Graphs with Ray` if that sounds better